R/lru.R

```diff
@@ -25,10 +30,15 @@ LRUcache_ <- R6::R6Class("LRUcache_",
         rm(list = name, envir = private$data)
       }
     },
-    last_accessed = function(name) {
+    last_get = function(name) {
       stopifnot(is.character(name) && length(name) == 1)
       stopifnot(name %in% ls(private$data))
-      private$peek(name)
+      private$data[[name]]$last_get
+    },
```

Comment:
Why not keep both `last_accessed` and `last_get`? Then you avoid making a breaking change.

Reply:
That's a good point, maybe `last_accessed` is a better name than `last_get`, might `s/accessed/get/g`. Will do another PR :)